iotdb-core/datanode/src/assembly/resources/conf/iotdb-datanode.properties

```diff
@@ -60,6 +60,19 @@ dn_data_region_consensus_port=10760
 # The time of data node waiting for the next retry to join into the cluster.
 # dn_join_cluster_retry_interval_ms=5000
 
+
+# is SSL enabled
+# enable_thrift_ssl=false
+
+# SSL key store path
+# key_store_path=
+
+# SSL key store password
+# key_store_pwd=
+
+# SSL client timeout (in millisecond)
+# client_timeout=300000
```

Comment:
What's the point of separating out a single parameter? Is there a problem if we use dn_connection_timeout?